todos-app/README.md

```diff
@@ -72,4 +72,29 @@ To set up both frontend and backend, simply run,
 docker compose up
 ```
 Wait a few seconds, and the frontend app should be up and running at http://localhost:8080/.  
-You can use the frontend app to navigate between pages and do basic CRUD operations on tasks.
\ No newline at end of file
+You can use the frontend app to navigate between pages and do basic CRUD operations on tasks.
+
+## Maintenance Mode
+### Backend API
+To have a service enter maintenance mode, execute a rake command,
+```bash
+# could be user-service
+docker exec -it task-service rake maintenance:start
+```
+
+To exit maintenance mode, 
+```bash
+# could be user-service
+docker exec -it task-service rake maintenance:end
+```
```

Comment:
~~Since there is two backend, maybe we should mention about it that we need to run two rake file.~~

oh, you wrote it at the commented out lines.